app/services/import_polygons.rb

```diff
@@ -14,7 +14,7 @@ def initialize(api_location_type:)
   end
 
   def call
-    response = JSON.parse(HTTParty.get(LOCATION_POLYGON_SETTINGS[api_location_type][:boundary_api]))
+    response = HTTParty.get(LOCATION_POLYGON_SETTINGS[api_location_type][:boundary_api])
```

Comment:
if you remove it from here, I guess you have to add it again later, as we still need to parse JSON

Reply:
The task still works locally!